src/__experimental__/components/textarea/documentation/info.js

```diff
@@ -8,7 +8,7 @@ const info = (
 
     <StoryHeader>Implementation</StoryHeader>
 
-    <StoryCode padded>import Textarea from {'carbon-react/lib/components/textarea'}</StoryCode>
+    <StoryCode padded>import Textarea from carbon-react/lib/components/textarea</StoryCode>
```

Comment:
no quotes here?

Reply:
will add them, I just ran `eslint --fix` 🤷‍♂

Reply:
fixed in https://github.com/Sage/carbon/commit/968a3e19e7973ece445801bd6c2a949beb923323